src/arkode/arkode_splittingstep.c

```diff
@@ -298,12 +321,21 @@ static int splittingStep_TakeStep(ARKodeMem ark_mem, sunrealtype* dsmPtr,
 
   for (int i = 1; i < coefficients->sequential_methods; i++)
   {
+    SUNLogInfo(ARK_LOGGER, "begin-sequential-method", "sequential method = %i",
+               i);
+
     N_VScale(ONE, ark_mem->yn, ark_mem->tempv1);
     retval = splittingStep_SequentialMethod(ark_mem, step_mem, i,
```

Comment:
Perhaps we can save this for a future PR, but I'd think that it would be good to print `ark_mem->tempv1` here when running with "extra debugging" enabled.